src/index.html

```diff
@@ -1,12 +1,314 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <div class="container header__content">
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        >
+
+        <ul class="menu">
+          <li class="menu__item menu__item--active">
+            <a class="menu__link menu__link--special " href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
```

Comment:
Add one empty line between sibling blocks inside the same parent block for both of them.
Same for all the similar cases